01.w_Defects/free_null_pointer.c

```diff
@@ -272,6 +273,7 @@ void free_null_pointer_009 ()
     free_null_pointer_009_func_001();
     for(i=0;i<5;i++)
     {
+        // TODO: this line dereferences a null pointer and program crashes before reaching line with error
```

Comment:
free_null_pointer_009dst is NULL at this point so deferencing it will segfault the program.